src/rapidcsv.h

```diff
@@ -54,7 +54,8 @@ namespace rapidcsv
           typeid(T) == typeid(unsigned long long) ||
           typeid(T) == typeid(float) ||
           typeid(T) == typeid(double) ||
-          typeid(T) == typeid(long double))
+          typeid(T) == typeid(long double) ||
+          typeid(T) == typeid(char))
```

Comment:
Performing std::to_string on a char converts the value in the char to its numeric representation, i.e. 'r' -> "114", which it seems is not what we want. Thus it needs special handling (not same as the numeric datatypes). Perhaps something like string(1, pVal).